.travis.yml

```diff
@@ -16,13 +16,12 @@ env:
     - PYTHON=""
   matrix:
     - CONDA_JL_VERSION="3"  # Documentation stage will use the first variable set
-    - CONDA_JL_VERSION="2"
```

Comment:
Sorry to be a pain but I think it's worth removing Conda 2 support as a separate PR

Reply:
ok I've reverted the change